src/commands/help.ts

```diff
@@ -0,0 +1,49 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as vscode from 'vscode';
+import { IActionContext, IAzureQuickPickOptions } from 'vscode-azureextensionui';
+
+import { ext } from '../extensionVariables';
+import { openExternal } from '../utils/openExternal';
+import { getDockerExtensionPackageJson } from '../utils/extension';
+
+interface HelpMenuItem extends vscode.QuickPickItem {
+    handler: () => Promise<void>
+}
+
+export async function showHelpMenu(context: IActionContext): Promise<void> {
+    let items: HelpMenuItem[] = [
+        { label: 'Get started with Docker...', handler: getStarted },
+        { label: 'Review Docker extension issues...', handler: reviewIssues },
+        { label: 'Report Docker extension issue...', handler: reportIssue },
+        { label: 'Edit settings...', handler: editSettings }
+    ];
+
+    const options: IAzureQuickPickOptions = { canPickMany: false, suppressPersistence: true }
+    let selectedItem: HelpMenuItem = await ext.ui.showQuickPick(items, options);
+    context.telemetry.properties.helpItem = selectedItem.handler.name;
```

Comment:
As the name effectively becomes a soft contract in telemetry analysis, I'd suggest using a constant so that future telemetry is unaffected by any refactoring (i.e. renaming) that might occur by someone not realizing where that name gets used.

Reply:
Good suggestion, fixed